themes-default/slim/views/history.mako

```diff
@@ -21,6 +21,56 @@ const startVue = () => {
         el: '#vue-wrap',
         data() {
             return {};
+        },
+        mounted() {
+            $('#historyTable:has(tbody tr)').tablesorter({
```

Comment:
Tries to load index.js.
GET http://localhost:8081/js/history/index.js?10032 404 (Not Found)

Reply:
From what I see, the import just needs to be removed from `main.mako`.
https://github.com/pymedusa/Medusa/blob/bca8f0c03547625646faec742988e192855e9d97/themes-default/slim/views/layouts/main.mako#L116

Reply:
@sharkykh @p0psicles the js was moved into that mounted method and I must have missed removing the script tag.